vocab_files/attribute_concepts/point-intercept-number.ttl

```diff
@@ -9,9 +9,13 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
     a skos:Concept ;
     dcterms:source "Ecological Field Monitoring protocols - Condition Module, draft v0.1, 30/11/2021" ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
+    skos:altLabel
+        "point intercept" ,
```

Comment:
Point intercept is usually referred to as the sampling method. It may be confusing to call this point intercept as an alternative label.

Reply:
See https://www.webpages.uidaho.edu/veg_measure/Modules/Lessons/Module%208(Cover)/8_3_Points.htm